walkthroughs/week-5-tdd/intro/src/test/java/com/google/sps/GreeterTest.java

```diff
@@ -30,4 +30,23 @@ public void testGreeting() {
 
     Assert.assertEquals("Hello Ada", greeting);
   }
+
```

Comment:
No action required, but just a note on a method like this and its tests. If the user submits a string where none of them are letters, like "^$@$*(@", then the method will return "Hello ". This is where we'd usually throw an error of some kind. If we did that, another test would make sure that an error was thrown with that input.